bin/release-build.sh

```diff
@@ -12,5 +12,10 @@ if [[ $# != 1 ]]; then
 fi
 
 log_file="/build/logs/${1}.log"
-nohup time ruby ${BUILD_DIR}/scripts/vmbuild.rb --type release --upload --reference $1 --copy-dir ${1%%-*} > $log_file 2>&1 &
+copy_dir="${1%%-*}"
+
+nohup sh -c \
+        "echo time ruby ${BUILD_DIR}/scripts/vmbuild.rb --type release --upload --reference $1 --copy-dir $copy_dir \
+         && time ruby ${BUILD_DIR}/scripts/vmpublish.rb --copy-dir $copy_dir" \
```

Comment:
I can see it both ways.  My main concern is that there is some complexity because the build code isn't handing off to the publish code so we have to find the right images, and do some things that the build code would just know.  

So, how about this... can we make the publish code separate but still offer the option to run it for all successful builds?  I think that's what this shell code change here is saying.  I'd rather the build code call publish code based on the results of each build.  We can add publish CLI code later once the interface for publish is done.